compiler/src/iree/compiler/Codegen/LLVMGPU/Passes.cpp

```diff
@@ -364,6 +369,13 @@ static void addGPUBufferizePasses(OpPassManager &funcPassManager) {
   };
   funcPassManager.addPass(
       createIREEComprehensiveBufferizePass(allocationFn, memcpyFn));
+
+  if (clUseDirectLoad) {
+    // Convert linalg.copy to direct loads. This has to be before any
+    // canonicalization.
+    funcPassManager.addPass(createGPULowerToGlobalLoadsPass());
```

Comment:
This pass doesn't need to be guarded by the flag since we do the lowering based on the attribute anyway.

Reply:
That is right. But since this option is only turned on in a few designated cases, I would say let's skip adding it into the pipeline in the rest of the cases to avoid unnecessary passes.